open-banking-accelerator/accelerators/ob-apim/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>com.wso2.openbanking.accelerator</groupId>
         <artifactId>open-banking-accelerator-modules</artifactId>
-        <version>3.2.39-SNAPSHOT</version>
```

Comment:
shallwe change PR title.IT's not bumping from 3.0 to 3.3.0,we r updating from 3.2.x to 3.3.0

Reply:
updated